src/ImageSharp/Formats/Jpeg/JpegSubsample.cs

```diff
@@ -8,6 +8,11 @@ namespace SixLabors.ImageSharp.Formats.Jpeg
     /// </summary>
     public enum JpegSubsample
     {
+        /// <summary>
+        /// Only luminance - No chrome channels.
+        /// </summary>
+        Grayscale,
```

Comment:
What happens if someone tries to use a `JpegEncoder` with `Subsample` being incompatible with `ColorType`?

Reply:
Subsample is only configured on the Chroma components in the current settings.

If in the future, there would be a Subsample for Luminance: Then ColorType takes precedence. See line 193 of JpegEncoderCore.cs